circuit_knitting/__init__.py

```diff
@@ -20,4 +20,4 @@
     # Package is not installed
     pass
 
-__all__ = []
+__all__: list[str] = []
```

Comment:
I was surprised the linter flagged this , do we do this other places?

Reply:
It's only because it's an empty list that the annotation is needed. I wanted to prevent the two imports from being re-exported, which is why I included this line.